scripts/dockerfiles/polkadot_injected_release.Dockerfile

```diff
@@ -25,15 +25,15 @@ RUN apt-get update && \
 		libssl1.1 \
 		ca-certificates \
 		gnupg && \
-	useradd -m -u 1000 -U -s /bin/sh -d /polkadot polkadot && \
+	useradd -m -u 1000 -U -s /bin/sh -d /polkadot polkadot
 # add repo's gpg keys and install the published polkadot binary
-	gpg --recv-keys --keyserver ${GPG_KEYSERVER} ${POLKADOT_GPGKEY} && \
+RUN gpg --keyserver ${GPG_KEYSERVER} --recv-keys ${POLKADOT_GPGKEY} && \
```

Comment:
Is it needed to divide one `RUN` into several? I thought all commands were in one `RUN` in order to have less layers

Reply:
No, i suppose i can join them again. I think i did that for debugging purposes.